server/models/services/CollectionJobService.js

```diff
@@ -585,13 +592,6 @@ const scheduleCollectionJob = async (
     transaction
   });
 
-  const atVersion = await getAtVersionWithRequirements(
```

Comment:
Given this removal, doesn't the scheduleCollectionJobResolver#L17 now need to also pass `atVersion`? Assigning a bot now immediately fails or is that intended as part of the follow-up?

Reply:
I didn't finish addressing the changes on the frontend and that is part of why I didn't want to merge this work until more had aggregated here. So, yes, it is meant to be a follow-up but the work shouldn't be merged without it

But, the point you have here raises a really good question that makes me wonder why the `scheduleCollectionJob` gql mutation in the integration test is still working. Will look into it

Reply:
This was still working because of the way that the test version of a collection job is initialized. Luckily no eerie mysteries this time.

I took the resolver to a slightly more logical stopping point for the frontend work with [5c01031](https://github.com/w3c/aria-at-app/pull/1320/commits/5c01031a3701f238cbbe6c3155f86c227577b926)